### ISSUE TITLE
Add a new configuration value of keepNewestVersionOnly or something similar

### ISSUE THREAD

**Issue by loganankarbergmdu** [pmckeown/dependency-track-maven-plugin#425]
Currently if the version of my project increments from 1.0 to 1.1 in a multi-module maven project, there are simply an entire set of new projects created with the new version, and the last versions do not get deleted. I would like to still show the actual version in dependency track, so manually setting it to "development" or something similar isn't a good solution for me.

I would also not like to just delete the old project and create a new one because then I lose all of the history that was tied to that project, as far as vulnerabilities go.

A new configuration to allow me to just enable keepNewestVersionOnly and that causes the verison to update, would be perfect.

**Comment by elmuerte** [comment]
I think this is better suited as a feature request for Dependency Track itself. It sounds similar to this request: https://github.com/DependencyTrack/dependency-track/issues/4532

Once the Dependency Track API exposes that functionality it could be added to the maven plugin.

**Comment by pmckeown** [comment]
Yeah agreed - let's see if this get implemented in the main application first and use that as the basis for adding this feature to the plugin.
